static/chrome/Dockerfile

```diff
@@ -22,7 +22,13 @@ COPY entrypoint.sh /
 RUN \
     chmod +x /usr/bin/chromedriver && \
     chown -R selenium:nogroup /etc/opt/chrome/policies/managed/
-USER selenium
+
+#Use root user
+USER root
+
+RUN mv /bin/uname /bin/uname.orig
+RUN printf '#!/bin/bash\n\nif [[ "$1" == "-r" ]] ;then\n echo '4.9.250'\n exit\nelse\n uname.orig "$@"\nfi' > /bin/uname
+RUN chmod 755 /bin/uname
```

Comment:
I found a similar workaround here: https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/tests/ci/docker_images/linux-x86/ubuntu-10.04_gcc-4.1x_corretto/Dockerfile#L10

Let's add a comment and a link to the glibc bug ticket: https://bugs.launchpad.net/ubuntu/+source/glibc/+bug/1962225